### ISSUE TITLE
Unknown filetypes ignored during extraction

### ISSUE THREAD

**Issue by zourtney** [wombleton/angular-gettext-loader#1]
I am unable to use this plugin to extract text from custom file sources. In my case, transpiled Typescript:

_webpack.config.js_

``` js
loaders: [
  {
    test: /\.ts$/,
    loaders: ['angular-gettext-extract', 'ts']
  },
  ...
```

**Comment by zourtney** [comment]
I did some digging. I see the `angular-gettext-tools` extraction library makes parsing assumptions based on file extension.

_angular-gettext-tools/lib/extract.js_

``` js
this.options = _.extend({
  ...
  extensions: {
    htm: 'html',
    html: 'html',
    php: 'html',
    phtml: 'html',
    tml: 'html',
    ejs: 'html',
    erb: 'html',
    js: 'js',
    tag: 'html',
    jsp: 'html'
  },
```

The extraction library ignores my `ts-loader` output -- though valid ES5 -- since it still carries the `.ts` extension.

Clearly `extensions` can be overridden, so I tried defining it via `&extensions={"ts":"js"}` query param. It fails (`loader-utils` seems to make no effort to parse objects, instead treating it as a string)

```
angular-gettext-extract?extensions={"ts:"js"}
```

To me, it seems like the simplest solution would be to add parsing logic for the `extensions` param, when defined. Pull request coming...

**Comment by zourtney** [comment]
Pull request created. I'm open to suggestions, if you have any better ideas.

**Comment by zourtney** [comment]
@wombleton, can we get this merged?

**Comment by wombleton** [comment]
@zourtney I thought I had — the PR is included in `v0.5.0`?

**Comment by zourtney** [comment]
Oh, my mistake -- yes, everything looks good. Thanks!
